app/actions/playlists.js

```diff
@@ -29,19 +30,41 @@ export function addPlaylist(tracks, name) {
   };
 }
 
+export function deletePlaylist(name) {
+  return dispatch => {
+    let playlists = store.get('playlists') || {};
+    if (playlists) {
+      for (let i = 0; i < playlists.length; i++) {
```

Comment:
Better to use lodash's `filter` function here: https://lodash.com/docs/4.17.11#filter
More explicit and shorter. Easier to see the intention too.

Reply:
Now I have used lodash's findIndex method to get the playlist index and then I am removing it from the array. How else the logic can be improved?